server/src/main/java/com/linecorp/centraldogma/server/metadata/RepositorySupport.java

```diff
@@ -116,16 +114,7 @@ CompletableFuture<Revision> push(String projectName, String repoName,
 
         return executor.execute(Command.transform(null, author, projectName, repoName, Revision.HEAD,
                                                   commitSummary, "", Markup.PLAINTEXT, transformer))
-                       .thenApply(CommitResult::revision)
-                       .exceptionally(cause -> {
-                           final Throwable peeled = Exceptions.peel(cause);
-                           if (peeled instanceof RedundantChangeException) {
```

Comment:
I've realized that we don't need this logic anymore after introducing the transformer in #1069 because it will raise an IllegalArgumentException if there's no change.

Reply:
I'm not sure `IllegalArgumentException` is a proper exception to understand the context.